Unmodded versions of Oceania, Black Sea and Storm for "Battle" section

CCCP84 opened the pull request:
This is a not-modified versions of my three best maps: Oceania, Black Sea and Storm.
All three maps are excellent. On my personal scale Oceania and the Black Sea are gold and the storm is silver.
I hope for your support.

CCCP84 committed:
My best 3 maps (unmodded versions for Battle section)

CCCP84 committed:
Update 8_black_sea.map

Version name change to 5.5

CCCP84 committed:
Update 8_oceania.map

Version name change to 5.5

CCCP84 committed:
Update 8_storm.map

Version name change to 5.5

CnCRAZER reviewed (changes requested):
I'm pretty sure this requires running **maps-updater.bat** within `tools\mpmaps-updater` to update the mpmaps.ini? @alexp8

CnCRAZER committed:
Update MPMaps.ini

CCCP84 committed:
Merge pull request #1 from CnCRAZER/unmodded-Oceania-Storm-BlackSea

Update MPMaps.ini

CCCP84 committed:
Update MPMaps.ini

Gamemodes=battle

CCCP84 committed:
Update 8_black_sea.map

CCCP84 committed:
Update 8_oceania.map

CCCP84 committed:
Update 8_storm.map

CCCP84 committed:
Update 8_black_sea.map

CCCP84 committed:
Update 8_oceania.map

CCCP84 committed:
Update 8_storm.map

CCCP84 committed:
Update 8_black_sea.map

Replaced events from "Elapsed Time" to "Any Event"
And some other minor changes

CCCP84 committed:
Update 8_oceania.map

Replaced events from "Elapsed Time" to "Any Event" And some other minor changes

CCCP84 committed:
Update 8_storm.map

Replaced events from "Elapsed Time" to "Any Event"

CCCP84 committed:
Update 8_black_sea.map

CCCP84 committed:
Update 8_black_sea.map

CCCP84 committed:
Update 8_oceania.map

CCCP84 committed:
Update 8_storm.map

CCCP84 committed:
Update 8_oceania.map

CCCP84 committed:
Update 8_oceania.map

alexp8 reviewed (changes requested):
This PR will need to be closed and re-created because the changes in MPMaps.ini are out of date.

Also the maps can go in `/Maps/Battle/`

Or DM me the maps on discord and I will create the pull request.
